### ISSUE TITLE
rename repos

### ISSUE THREAD

**Issue by agarwal** [andrewray/iocaml#28]
Right now you have the following mappings:

opam package | github repo | ocamlfind package
------------------   |  -------------- | -------------------------
iocaml | iocamlserver | none, executable only
iocaml-kernel | iocaml | iocaml-kernel & iocaml-kernel.notebook
iocamljs-kernel | iocamljs | iocamljs.kernel

The first two rows are rather confusing because `iocaml` shows up in both. Your opam package names seem the saner ones, so I'd suggest renaming the GitHub repo iocaml -> iocaml-kernel. That would resolve the ambiguity. You could also simplify the name of the GitHub repo iocamlserver to iocaml for the sake of consistency with its opam package name, but this is less important.

**Comment by andrewray** [comment]
Yes, it's a very fair point.  The various bits were evolved rather than designed, hence it's a bit of a mess!

I've thought about a tidy up and I think I would eventually probably just collapse the 3 bits into one repo in the end.
